src/blockchains/eth/eth_worker.ts

```diff
@@ -106,11 +112,11 @@ export class ETHWorker extends BaseWorker {
     const [traces, blocks, receipts] = await this.fetchData(fromBlock, toBlock)
     const events: (ETHTransfer | EOB)[] = this.transformPastEvents(fromBlock, toBlock, traces, blocks, receipts)
     events.sort(transactionOrder)
-    if (this.settings.CHECK_QUALITY) {
-      checkETHTransfersQuality(events, fromBlock, toBlock)
+    if (this.ethClientVerification) {
```

Comment:
The main task in this PR - use verification node. Here we use the presence of a verification node URL as indicator to do QC.